services/document-manager/backend/app/__init__.py

```diff
@@ -59,7 +61,28 @@ def register_extensions(app):
     db.init_app(app)
     jwt.init_app(app)
     migrate.init_app(app, db)
-
     CORS(app)
 
     return None
+
+
+def make_celery(app=None):
```

Comment:
Does this celery related code work / is it used? I thought Celery wasn't working, and if not I question whether we should commit this code to our code base.

Reply:
Yup! It works and the tasks are dependant on it. There were just two minor issues with it running on OpenShift. The first one is OpenShift not wanting to start it as a background task (so we just have to launch the command manually) and second is that the concurrency can only be 1 (because OpenShift seems to immediately kill the other processes when we start Celery with more threads).